server/app/views/applicant/ProgramIndexView.java

```diff
@@ -367,6 +366,21 @@ private LiTag programCard(
         .with(actionDiv);
   }
 
+  /**
+   * If eligibility is gating, the eligibility tag should always show when present. If eligibility
+   * is non-gating, the eligibility tag should only show if the user may be eligible.
+   */
+  private boolean shouldShowEligibilityTag(
+      Http.Request request, ApplicantService.ApplicantProgramData cardData) {
+    if (!featureFlags.isProgramEligibilityConditionsEnabled(request)
```

Comment:
nit: These conditions don't feel related or dependent on each other. Can we break them apart into

```java
if (!featureFlags.isProgramEligibilityConditionsEnabled(request) {
    return false;
}

if(!cardData.isProgramMaybeEligible().isPresent()) {
    return false;
}
```
so it's a little easier to parse?

Reply:
Lol that's how I had it originally but then thought maybe that made it more confusing. Helpful feedback that that is in fact easier to parse :)